miso-web/src/main/webapp/scripts/hot_sample.js

```diff
@@ -584,93 +585,91 @@ HotTarget.sample = (function() {
     },
 
     getBulkActions: function(config) {
-      return [
-          {
-            name: "Edit",
-            action: function(samples) {
-
-              if (samples.some(function(sample) {
-                return sample.sampleClassId;
-              }) && !Constants.isDetailedSample) {
-                alert("There's detailed samples, but MISO is not configured for this.");
-                return;
-              }
-
-              var classes = getSampleClasses(samples);
-              var classesAliases = Utils.array.deduplicateString(classes.map(function(sampleClass) {
-                return sampleClass.alias;
-              }));
-              if (classesAliases.length > 1) {
-                alert("You have selected samples of classes " + classesAliases.join(" & ")
-                    + ". Please select samples from only one class.");
-                return;
-              }
-
-              window.location = "/miso/sample/bulk/edit?" + jQuery.param({
-                ids: samples.map(Utils.array.getId).join(',')
-              });
-            }
-
-          }, {
-            name: "Propagate",
-            action: function(samples) {
-              HotUtils.warnIfConsentRevoked(samples, function() {
-                var idsString = samples.map(Utils.array.getId).join(",");
-                var classes = getSampleClasses(samples);
-
-                // In the case of plain samples, this will be empty, which is fine.
-                var targets = getChildSampleClasses(classes).sort(Utils.sorting.sampleClassComparator).map(function(sampleClass) {
-
-                  return {
-                    name: sampleClass.alias,
-                    action: function(replicates) {
-                      window.location = "/miso/sample/bulk/propagate?" + jQuery.param({
-                        parentIds: idsString,
-                        replicates: replicates,
-                        sampleClassId: sampleClass.id
-                      });
-                    }
-                  };
+      return [{
+        name: "Edit",
+        action: function(samples) {
+
+          if (samples.some(function(sample) {
+            return sample.sampleClassId;
+          }) && !Constants.isDetailedSample) {
+            alert("There's detailed samples, but MISO is not configured for this.");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this usage of alert(...). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1442)